Tests/PointFreeTests/DatabaseTests.swift

```diff
@@ -81,4 +80,69 @@ final class DatabaseTests: LiveDatabaseTestCase {
 
     XCTAssertEqual(subscription.id, freshUser.subscriptionId)
   }
+
+  func testUpdateEpisodeProgress() {
```

Comment:
this test runs against the live DB to make sure that the query logic around taking the max of percentages works correctly.